db/schema.rb

```diff
@@ -10,10 +10,32 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2023_02_24_213052) do
+ActiveRecord::Schema[7.1].define(version: 2024_05_19_210340) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "application_pets", force: :cascade do |t|
+    t.bigint "application_id", null: false
+    t.bigint "pet_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["application_id"], name: "index_application_pets_on_application_id"
+    t.index ["pet_id"], name: "index_application_pets_on_pet_id"
+  end
```

Comment:
Your join table should have a `status` column, this is the only way to properly implement US 12, 13, and 14.